tools/example-agent.yml

```
@@ -0,0 +1,73 @@
+#     type: stdio
+#     command: npx
+#     args:
+#       - -y
+#       - "@modelcontextprotocol/server-filesystem"
+#       - .
+
+# Custom tools configuration
+customTools:
+  # Enable all tools (default behavior)
+  enabledTools: 'all'
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)